simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFeedFragment.kt

```diff
@@ -852,11 +852,14 @@ class CatalogFeedFragment : Fragment(R.layout.feed), AgeGateDialog.BirthYearSele
       button.setButtonDrawable(R.drawable.catalog_facet_tab_button_background)
 
       button.text = facet.title
-      button.setTextColor(ContextCompat.getColor(this.requireContext(), R.color.neutralColorPrimary))
+      button.setTextColor(
+        ContextCompat.getColor(this.requireContext(), R.color.simplified_button_text)
+      )
       button.setOnClickListener {
         this.logger.debug("selected entry point facet: {}", facet.title)
         this.viewModel.openFacet(facet)
       }
+      button.setPadding(0, 0, 0, 0)
```

Comment:
I think you can use the `setPadding(0)` from the Kotlin utils to simplify this.

Reply:
Thanks. I've updated this.